### PR TITLE
Move B*,C* tests from NotYetReady to Ready

### PR DESCRIPTION
**Pull request by CarlosFigueiraMSFT**
Continuing to move tests from the "NotYetReady" folder to the place where they can actually run. Files starting with B, and C. Also update how time values are compared in test results.

### CODE REVIEW

File: src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Coalesce.txt [hunk 1]
```diff
@@ -8,12 +8,24 @@
...
+"a"
+
 >> Coalesce( Blank(), "", Blank(), "", "3", "4" )
 "3"
 
+>> Coalesce(Blank(), "", Blank(), "", Blank(), " ", "3", "4")
+" "
+
 >> Coalesce("")
 Blank()
```
Comment (Ananya-Gupta24) [inline src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Coalesce.txt:25]: Unrelated to your changes, could you add a case here for an input with only Blanks and empty strings and verify that it returns Blank()?

Comment (CarlosFigueiraMSFT) [reply src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Coalesce.txt:26]: Added

File: src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Count.txt [hunk 2]
```diff
@@ -1,20 +1,76 @@
...
+
+>> Count(Table({a:Blank()},{a:Blank()},{a:Blank()}))
+0
+
+// ******** BLANK TABLE ********
+
+>> Count(If(1<0,[1,2,3]))
+0
+
+// ******** ERRORS ********
```
Comment (Ananya-Gupta24) [inline src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Count.txt:60]: Should we have error cases here for when the first argument is an empty string or Blank() followed by a string or number value like Count(["", 1]) and Count([Blank(), "1"])?

Comment (CarlosFigueiraMSFT) [reply src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Count.txt:60]: Good point, will add some cases for (compiler) errors where the table doesn't have a numeric column (which is the case in your examples)

**Review by Ananya-Gupta24 — APPROVED** [review]

**Review by Ananya-Gupta24 — APPROVED** [review]

**Pull request merged**